FreeRTOS/FreeRTOS
# feat: add MQTT Mutual Auth on QEMU demo

Add MQTT Mutual Auth on QEMU demo

Description
-----------
Adds a demo for MQTT Mutual Auth on QEMU.

Test Steps
-----------
n/a (see README.md in demo folder for instructions to build and test)

Checklist:
----------



Related Issue
-----------
n/a


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
